signal_analog/util.py

```diff
@@ -23,6 +25,22 @@ def in_given_enum(value, enum):
         raise ValueError(msg.format(value, valid_values))
 
 
+def check_markdown(text, error_message=None):
+    """Determines if the given text is a markdown and able to generate HTML
+    """
+    if text is '' or text is None:
+        if error_message:
```

Comment:
Does the if statement for the error message change functionality at all?
is ValueError()  different than ValueError(None)?

Reply:
no, doesn't change in the functionality.

Reply:
This is _really_ subjective feedback, but `ValueError(None)` is just a tad noisier than just `ValueError()`. This is what we're doing on a few other util methods so I'm inclined to leave it as is.

I think in the future it would be great to generate more than just ValueError similar to what we have in the `signal_analog.errors` module. No concrete plans there yet, though!